uiowa/uiowa
# [Bug] Admissions upcoming events

# How to test


`ddev blt ds --site=admissions.uiowa.edu`
Check for a clean sync
Add some events, including upcoming and past
Check out https://admissions.uiowa.ddev.site/events
See that upcoming events are displayed
Check that the filters visible are still limited to those used in prod at admissions.uiowa.edu/events
Check that https://admissions.uiowa.ddev.site/events/calendar has all the proper events displaying
